easy-ui-react/src/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { ReactNode } from "react";
+import { Key } from "react-aria";
+import { AriaLabelingProps } from "@react-types/shared";
+import { ToggleButtonGroup, ToggleButton } from "react-aria-components";
+import { classNames, getComponentThemeToken } from "../utilities/css";
+import { ThemeColorAliases } from "../types";
+import { Text } from "../Text";
+
+import styles from "./RadioButtonGroup.module.scss";
+
+export type RadioButtonGroupButtonProps = {
+  /**
+   * The label for the radio button.
+   */
+  children: ReactNode;
+  /**
+   * Whether the radio button is disabled or not.
+   */
+  isDisabled?: boolean;
+  /**
+   * An identifier for the item in selectedKeys.
+   */
+  id: string;
+};
+
+function RadioButtonGroupButton(props: RadioButtonGroupButtonProps) {
+  const { children, isDisabled, id } = props;
+
+  return (
+    <ToggleButton
+      className={classNames(styles.RadioButtonGroupButton)}
+      isDisabled={isDisabled}
+      id={id}
+    >
+      <Text variant="small_button">{children}</Text>
+    </ToggleButton>
+  );
+}
+
+export type RadioButtonGroupProps = AriaLabelingProps & {
+  /**
+   * Color for the selected button in the group.
+   * @default "primary.500"
+   */
+  color?: ThemeColorAliases;
+  /**
+   * Whether single or multiple selection is enabled.
+   * @default "single"
+   */
+  selectionMode?: "single" | "multiple";
+  /**
+   * Whether the collection allows empty selection.
+   */
+  disallowEmptySelection?: boolean;
+  /**
+   * 	The currently selected keys in the collection (controlled).
+   */
+  selectedKeys?: Set<Key>;
+  /**
+   * The initial selected keys in the collection (uncontrolled).
+   */
+  defaultSelectedKeys?: string[];
+  /**
+   * Whether all items are disabled.
+   */
+  isDisabled?: boolean;
+  /**
+   * RadioButtonGroup Buttons
+   */
+  children: ReactNode;
+  /**
+   * Handler that is called when the selection changes.
+   */
+  onSelectionChange?: (keys: Set<Key>) => void;
+};
+
+export function RadioButtonGroup(props: RadioButtonGroupProps) {
```

Comment:
can you add a tsdoc comment for this? if you look at `RadioGroup` component, you'll see an example tsdoc block comment. this helps developers when using the component in an IDE.

this is what the "TSDoc spec for API surface area" refers to in the pull request template